force_control_node:
  rate: 100
  pid: {kp: 0.1974, ki: 0.0, kd: 0.544}
  mass: 0.01
  sync: true
  wrench_topic: /force_control/command/wrench
  odom_topic: /force_control/odometry
  ref_topic: /force_control/command/position